CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ description: |-
 - Add vkcs_networking_sdn data source for getting a list of available SDNs.
 - Add traffic_selector_ep_merge argument to vkcs_vpnaas_site_connection resource.
 - Add external_fixed_ips computed field to vkcs_networking_router resource and data source.
+- Up compute api version to 2.42.
+- Fix searching subnet by tags.
```

Comment:
Fix searching for a subnet with a filter by "tags" in the vkcs_networking_subnet data source.